File: backend/app/modules/benson_edition/manifest.py
from app.core.edition import is_benson_edition
from app.core.module_loader import ModuleManifest

manifest = ModuleManifest(
    name="benson_edition",
    version="0.1.0",
    display_name="Benson Edition",
    description="Integrated Benson edition overlay for Eastern Oregon operations.",
    author="Benson Home Solutions",
    category="integration",
    depends=[
        "benson_operations",
        "benson_workforce",
        "benson_finance",
        "benson_security",
        "benson_workers",
    ],
    auto_install=True,
    enabled=is_benson_edition(),
)

File: frontend/src/modules/benson-operations/manifest.ts
import { lazy } from 'react';
import { Building2 } from 'lucide-react';
import { isBensonEdition } from '@/editions/config';
import type { ModuleManifest } from '../_types';

const BensonOperationsPage = lazy(() =>
  import('@/features/benson/BensonOperationsPage').then((module) => ({
    default: module.BensonOperationsPage,
  })),
);

export const manifest: ModuleManifest = {
  id: 'benson-operations',
  name: 'modules.benson.name',
  description: 'modules.benson.description',
  version: '12.9.0',
  icon: Building2,
  category: 'tools',
  defaultEnabled: isBensonEdition,
  routes: [{ path: '/benson', title: 'Benson Operations', component: BensonOperationsPage }],
  navItems: [
    {
      labelKey: 'nav.benson_operations',
      to: '/benson',
      icon: Building2,
      group: 'tools',
    },
  ],
  searchEntries: [
    {
      label: 'Benson Operations',
      path: '/benson',
      keywords: ['benson', 'eastern oregon', 'operations', 'jurisdiction'],
    },
  ],
  translations: {
    en: {
      'modules.benson.name': 'Benson Operations',
      'modules.benson.description': 'Benson edition operating profile and integrated workflows',
      'nav.benson_operations': 'Benson Operations',
    },
  },
};
